pubpub/pubpub
# feat: include release data in analytics

- feat: add release info for pubs in analytics payload
- feat: send collectionIds as string instead of array
- fix: Update getRelease logic to correctly capture release from path
- feat: include release with analytics data

## Issue(s) Resolved

Not possible to see which release was visited in the analytics and `analytics` sends the canonical url instead of the actual visited url.

## Test Plan
1. Go to any released pub
2. Check if the event to `/api/analytics/track` includes a `release: \<number\>`
3. Go to the draft
4. Check if the event to `/api/analytics/track` includes a `release: "draft"`

It will not validate correctly but that's bc the canonical-url for qubqub is not a valid url.
## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
